Replace /var/run with /run as per the new FHS.

--- dmd.texi
+++ dmd.texi
@@ -154,7 +154,7 @@
 As its name suggests, dmd is just a daemon that (usually) runs in the
 background, so you will not interact with it directly.  After it is
 started, dmd will listen on a socket special file, usually
-@code{/var/run/dmd/socket}, for further commands.  You use the tool
+@code{/run/dmd/socket}, for further commands.  You use the tool
 @dfn{deco} to send these commands to dmd.  Usage of deco is simple and
 straightforward: To start a service called @code{apache}, you use:
 
@@ -408,7 +408,7 @@
 @itemx --socket=@var{file}
 @cindex socket special file
 Receive further commands on the socket special file @var{file}.  If
-this option is not specified, @file{@var{localstatedir}/run/dmd/socket} is
+this option is not specified, @file{/run/dmd/socket} is
 taken.
 
 If @code{-} is specified as file name, commands will be read from
@@ -451,7 +451,7 @@
 @item -s @var{file}
 @itemx --socket=@var{file}
 Send commands to the socket special file @var{file}. If this option is
-not specified, @file{@var{localstatedir}/run/dmd/socket} is taken.
+not specified, @file{/run/dmd/socket} is taken.
 
 @end table
 
@@ -477,7 +477,7 @@
 @item -s @var{file}
 @itemx --socket=@var{file}
 Send commands to the socket special file @var{file}. If this option is
-not specified, @file{@var{localstatedir}/run/dmd/socket} is taken.
+not specified, @file{/run/dmd/socket} is taken.
 
 @end table
 
@@ -503,7 +503,7 @@
 @item -s @var{file}
 @itemx --socket=@var{file}
 Send commands to the socket special file @var{file}. If this option is
-not specified, @file{@var{localstatedir}/run/dmd/socket} is taken.
+not specified, @file{/run/dmd/socket} is taken.
 
 @end table
 
--- modules/dmd/support.scm
+++ modules/dmd/support.scm
@@ -227,7 +227,7 @@
 ;; The directory where the socket resides.
 (define default-socket-dir
   (if (zero? (getuid))
-      (string-append %localstatedir "/run/dmd")
+      (string-append "/run/dmd")
       (string-append user-dmddir "/run")))
 
 ;; Unix domain socket for receiving commands in dmd.
